charts/consul/test/unit/controller-deployment.bats

```diff
@@ -46,18 +46,18 @@ load _helpers
 #--------------------------------------------------------------------
 # global.acls.manageSystemACLs
 
-@test "controller/Deployment: CONSUL_HTTP_TOKEN env variable created when global.acls.manageSystemACLs=true" {
```

Comment:
This replaces an old test which checked for an env variable that was backed by a kube secret (which this PR removes)